app/models/bulk_download.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+require 'zip'
+
+##
+# This class is used to create a zip file
+# of all the files in a FileSet
+class BulkDownload
+  attr_accessor :zip_file_name, :files
+  ##
+  # @param [String] ID for an ActiveFedora::Base work with FileSets
+  # This zips up all the files in a FileSet object
+  def initialize(work_id)
+    work = ActiveFedora::Base.find(work_id)
+
+    @files = work.file_sets.map { |file_set| file_set.files }
```

Comment:
Will this be an array of arrays?  Do we need to call `flatten` on it?

Reply:
Ah, I know, there's an `original_file` method.  So I think you can do something like this:
`@files = work.file_sets.map { |file_set| file_set.original_file }`.  Then you don't have to flatten or worry about derivatives